card/mod/05_standard/set/type/session.rb

```diff
@@ -0,0 +1,26 @@
+
+event :store_in_session, :after=>:approve, :on=>:create do
+  Env.session[key] = db_content
+  self.db_content = ''
+end
+
+event :update_in_session, :after=>:approve, :on=>:update do
+  if db_content_changed?
+    Env.session[key] = db_content
+    self.db_content = ''
+  end
+end
+
+event :delete_in_session, :after=>:approve, :on=>:delete do
+  Env.session[key] = nil
+  abort :success
```

Comment:
I suppose it begs the question of how you would ever delete the card itself.

I don't feel strongly (and wouldn't want to hold anything up for this), but I guess I'm tempted to say we should go ahead and delete it unless there's a good reason to have the card around.